app/lib/logic/rolling_updates.rb

```diff
@@ -359,9 +359,7 @@ module Controller
         helper_method :provider_can_use?
         private :provider_can_use?
       end
-
-      private
```

Comment:
If you do https://github.com/3scale/porta/pull/1258/files#r330067462 , you won't need to remove this :smile: